client/src/app/pages/about/about.component.html

```diff
@@ -0,0 +1,72 @@
+<app-page-heading>
+  <div heading>About Helios</div>
+  <div description>
+    <br />
+    Helios is an interactive platform designed to improve the efficiency and usability of CI/CD workflows in GitHub Actions. The platform enables better visualization, environment
+    tracking, deployment management and test analytics.
+  </div>
+</app-page-heading>
+
+<div class="container mx-auto p-6">
+  <div class="grid grid-cols-1 md:grid-cols-2 gap-6">
+    <div class="p-4 border rounded-lg shadow-md">
+      <h2 class="text-lg font-semibold mb-2">Main Features</h2>
+      <ul class="list-disc pl-5">
+        <li>Visualization of GitHub Actions pipelines</li>
+        <li>Environment locking/unlocking</li>
+        <li>Deployment management</li>
+        <li>Test analytics (coming soon)</li>
+      </ul>
+    </div>
+
+    <div class="p-4 border rounded-lg shadow-md">
+      <h2 class="text-lg font-semibold mb-2">Get in Touch</h2>
+      <p><a href="https://github.com/ls1intum/Helios/issues/new?template=1-bug-report.yml" class="text-blue-500" target="_blank">Report a Bug</a></p>
+      <p><a href="https://github.com/ls1intum/Helios/issues/new?template=2-feature-request.yml" class="text-blue-500" target="_blank">Request a Feature</a></p>
+      <p><a href="https://github.com/ls1intum/Helios/releases" class="text-blue-500" target="_blank">Release Notes</a></p>
+    </div>
+  </div>
+
+  <div class="mt-6 p-4 border rounded-lg shadow-md text-center">
+    <h2 class="text-lg font-semibold mb-2">Project Advisors</h2>
+    <div class="flex justify-center">
+      <div class="p-2 border rounded-md text-center">
+        <img [ngSrc]="'assets/about/krusche.jpg'" alt="Stephan Krusche" width="112" height="112" class="w-28 h-28 mx-auto rounded-full" />
```

Comment:
question: maybe we should fetch the images from github instead, because this site is rarely visited by a user and the images are increasing the bundle size.